### ISSUE TITLE
byte-code: Args out of range: "", 0, 

### ISSUE THREAD

**Issue by totallymike** [magit/magit#1739]
I have a repo where opening magit-status produces this error.  I can't, unfortunately, share the repo, and I'm not sure how to reproduce.  How can I get more information to help reproduce?

This is on the `next` branch

**Comment by totallymike** [comment]
For the record, toggling debug-on-error produces this stack trace:

There are a couple instances of `<repo-path>` where there used to be the absolute path of my repository.  I swapped that out when I pasted in this stack trace.

```lisp
Debugger entered--Lisp error: (args-out-of-range "" 0 7)
  match-string(1 "")
  byte-code("\204	\306H\n>\204\307\310\311\312#\210	\211\313\314I\210)\315\316\317\320\"\321\f\322\323#\324\2050\321\322\325#\324P
\326&\210@\204C\327!\211@\203\352\330\331@\"\206L\332A\333\334A\"\210AB\335\313B\"\335\336B\"\337\340\300@\341 \342\211\211\211\211\211\211C\342&
\211D\306H\n>\204\207\307\310\343\312#\210D\211E\344\345\346D\"\211F\203\237F\347=\202\303G\205\254\350\351D!G\"\211\205\302	\306H\n>\204\277\307\310\343\312#\210	\344H))I\210)D\211G\206\333C?\205\333HDHGCI\352\353\215\210D\210.\354\355!\210\326c\210\356	!\210\357	\306H\n>\204\307\310\360\312#\210	\361H\362\"\210	\306H\n>\204\307\310\363\312#\210	\211J\364\341 I)K\365\366\367	\306H\n>\2044\307\310\311\312#\210	\313H\"!L\370L!\205ELJL\212	\306H\n>\204V\307\310\360\312#\210	\361Hb\210`KW\203\222\371`\312\"\206kKM\372`\312\"\204\212\373`M\312	$\210L\203\212\373`M\374L$\210Mb\210)\202[,	H=\203\241\375	!\210\202	\306H\n>\204\257\307\310\376\312#\210	\377H\306H\n>\204\301\307\310\201O\312#\210	\306H\n>\204\317\307\310\376\312#\210	\377H\211N\201P	\306H\n>\204\346\307\310\376\312#\210	\377H\306H\n>\204\370\307\310\201O\312#\210	\306H\n>\204\307\310\376\312#\210	\377H\201PH	C\244I\210)\342\207" [branch it cl-struct-magit-section-tags v hash msg 0 error "%s accessing a non-%s" magit-section-type magit-section 1 commit magit-insert-heading magit-string-pad "Head: " 10 propertize face magit-hash " " magit-branch-local "\n" magit-get-tracked-branch magit-rev-format "%h %s" "" string-match "^\\([^ ]+\\) \\(.*\\)" match-string 2 vector cl-struct-magit-section point-marker nil magit-section-hidden 6 run-hook-with-args-until-success magit-section-set-visibility-hook hide magit-get-section magit-section-ident cancel-section (byte-code "\306\307\310\311\"\203\312\313\314#\202\315\316\317\320	\321#\205\322\312\n\313\323\320	\324#\325\230\203-\326\202.\327#\316\330\260!\210\331\f!\210\332\f\333H
>\204I\334\335\336\337#\210\f\340H\341\"\210\f\333H
>\204]\334\335\342\337#\210\f\2114\343\344 I)5\345\346\347\f\333H
>\204y\334\335\350\337#\210\f\351H\"!6\3526!\205\2126J6\212\f\333H
>\204\233\334\335\336\337#\210\f\340Hb\210`5W\203\327\353`\337\"\206\26057\354`\337\"\204\317\355`7\337\f$\2106\203\317\355`7\3566$\2107b\210)\202\240,\f8=\203\346\357\f!\210\202P\f\333H
>\204\364\334\335\360\337#\210\f\361H\333H
>\204\334\335\362\337#\210\f\333H
>\204\334\335\360\337#\210\f\361H\2119\363\f\333H
>\204'\334\335\360\337#\210\f\361H\333H
>\2047\334\335\362\337#\210\f\333H
>\204E\334\335\360\337#\210\f\361H\363H\fC\244I\210)\306\207" [hash branch upstream msg section89307 cl-struct-magit-section-tags magit-insert magit-string-pad "Upstream: " 10 propertize face magit-hash "missing" " " magit-get-boolean "branch" "rebase" "onto " magit-get "remote" "." magit-branch-local magit-branch-remote "\n" magit-insert-child-count set-marker-insertion-type 0 error "%s accessing a non-%s" magit-section-start magit-section 3 t magit-section-end 5 point-marker intern format "magit-%s-section-map" magit-section-type 1 boundp next-single-property-change get-text-property put-text-property keymap magit-section-show magit-section-parent 11 ...] 13) run-hooks magit-status-headers-hook magit-insert-child-count set-marker-insertion-type magit-section-start 3 ...] 15)
  magit-insert-status-headers()
  run-hooks(magit-status-sections-hook)
  byte-code("\306\307!\210\310!\210\311\312H	>\204\313\314\315\316#\210\317H\320\"\210\312H	>\204+\313\314\321\316#\210\211\322\323 I)\324\325\326\312H	>\204E\313\314\327\316#\210\330H\"!\331\f!\205S\fJ\212\312H	>\204c\313\314\315\316#\210\317Hb\210`W\203\227\332`\316\"\206v\333`\316\"\204\220\334`
\316$\210\f\203\220\334`
\335\f$\210
b\210)\202h,#=\203\246\336!\210\202\312H	>\204\264\313\314\337\316#\210\340H\312H	>\204\304\313\314\341\316#\210\312H	>\204\322\313\314\337\316#\210\340H\211$\342\312H	>\204\347\313\314\337\316#\210\340H\312H	>\204\367\313\314\341\316#\210\312H	>\204\313\314\337\316#\210\340H\342HC\244I\210)\306\207" [section89304 cl-struct-magit-section-tags v end map next run-hooks magit-status-sections-hook magit-insert-child-count set-marker-insertion-type 0 error "%s accessing a non-%s" magit-section-start magit-section 3 t magit-section-end 5 point-marker intern format "magit-%s-section-map" magit-section-type 1 boundp next-single-property-change get-text-property put-text-property keymap magit-section-show magit-section-parent 11 magit-section-children 12 magit-root-section v] 8)
  magit-status-refresh-buffer()
  apply(magit-status-refresh-buffer nil)
  magit-refresh-buffer()
  magit-mode-init("<repo-path>" magit-status-mode magit-status-refresh-buffer)
  apply(magit-mode-init "<repo-path>" (magit-status-mode magit-status-refresh-buffer))
  magit-status-internal("<repo-path>")
  magit-status(nil)
  call-interactively(magit-status nil nil)
  command-execute(magit-status)
```

**Comment by totallymike** [comment]
Also, it does bring me into a status buffer, where it shows the HEAD commit, but nothing else.  No changes, index, etc.

**Comment by vanicat** [comment]
Hello. Could you remove the *.elc that are in the magit directory, and re test it with debug-on-error on, so that the backtrace is more readable.

By the way what version on magit are you using?

**Comment by totallymike** [comment]
Sure thing.  I'm using the next branch, specifically commit `0b65528`

Here's a new backtrace:

```lisp
Debugger entered--Lisp error: (args-out-of-range "" 0 7)
  match-string(1 "")
  (let ((hash (match-string 1 string82716)) (msg (match-string 2 string82716))) (let* ((section82717 (progn (vector (quote cl-struct-magit-section) (quote branch) upstream (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp section82717) (>= (length section82717) 13) (memq (aref section82717 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v section82717)) (aset v 6 (let ((value ...)) (if value (eq value ...) (let ... ...)))))) (let ((magit-insert-section--current section82717) (magit-insert-section--parent section82717) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section ...))))) (catch (quote cancel-section) (magit-insert (concat (magit-string-pad "Upstream: " 10) (if hash (propertize hash ... ...) "missing") " " (and (magit-get-boolean "branch" branch "rebase") "onto ") (propertize upstream (quote face) (if ... ... ...)) " " msg "\n")) (magit-insert-child-count section82717) (set-marker-insertion-type (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (aref section82717 3)) t) (let* ((end (progn ... ...)) (map (intern ...)) (map (and ... ...))) (save-excursion (goto-char (progn ... ...)) (while (< ... end) (let ... ... ...)))) (if (eq section82717 magit-root-section) (magit-section-show section82717) (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 12 ...))))) section82717)))
  (let ((string82716 line)) (let ((hash (match-string 1 string82716)) (msg (match-string 2 string82716))) (let* ((section82717 (progn (vector (quote cl-struct-magit-section) (quote branch) upstream (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp section82717) (>= (length section82717) 13) (memq (aref section82717 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v section82717)) (aset v 6 (let (...) (if value ... ...))))) (let ((magit-insert-section--current section82717) (magit-insert-section--parent section82717) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil ...)))) (catch (quote cancel-section) (magit-insert (concat (magit-string-pad "Upstream: " 10) (if hash ... "missing") " " (and ... "onto ") (propertize upstream ... ...) " " msg "\n")) (magit-insert-child-count section82717) (set-marker-insertion-type (progn (or ... ...) (aref section82717 3)) t) (let* ((end ...) (map ...) (map ...)) (save-excursion (goto-char ...) (while ... ...))) (if (eq section82717 magit-root-section) (magit-section-show section82717) (progn (or ... ...) (let* ... ...)))) section82717))))
  (progn (setq line (or (magit-rev-format "%h %s" upstream) "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82716 line)) (let ((hash (match-string 1 string82716)) (msg (match-string 2 string82716))) (let* ((section82717 (progn (vector ... ... upstream ... nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp section82717) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v section82717)) (aset v 6 (let ... ...)))) (let ((magit-insert-section--current section82717) (magit-insert-section--parent section82717) (magit-insert-section--oldroot (or magit-insert-section--oldroot ...))) (catch (quote cancel-section) (magit-insert (concat ... ... " " ... ... " " msg "\n")) (magit-insert-child-count section82717) (set-marker-insertion-type (progn ... ...) t) (let* (... ... ...) (save-excursion ... ...)) (if (eq section82717 magit-root-section) (magit-section-show section82717) (progn ... ...))) section82717)))))
  (if (or upstream (setq upstream (magit-get-tracked-branch branch))) (progn (setq line (or (magit-rev-format "%h %s" upstream) "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82716 line)) (let ((hash (match-string 1 string82716)) (msg (match-string 2 string82716))) (let* ((section82717 (progn ...))) (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 6 ...))) (let ((magit-insert-section--current section82717) (magit-insert-section--parent section82717) (magit-insert-section--oldroot ...)) (catch (quote cancel-section) (magit-insert ...) (magit-insert-child-count section82717) (set-marker-insertion-type ... t) (let* ... ...) (if ... ... ...)) section82717))))))
  (catch (quote cancel-section) (if branch nil (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-type) (quote magit-section))) (let* ((v it)) (aset v 1 (quote commit))))) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash (quote face) (quote magit-hash)) " " (and branch (concat (propertize branch (quote face) (quote magit-branch-local)) " ")) msg "\n") (if (or upstream (setq upstream (magit-get-tracked-branch branch))) (progn (setq line (or (magit-rev-format "%h %s" upstream) "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82716 line)) (let ((hash (match-string 1 string82716)) (msg (match-string 2 string82716))) (let* ((section82717 ...)) (progn (or ... ...) (let* ... ...)) (let (... ... ...) (catch ... ... ... ... ... ...) section82717)))))) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref it 3)) t) (let* ((end (progn (or (and (vectorp it) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-end) (quote magit-section))) (let* ((v it)) (aset v 5 (point-marker))))) (map (intern (format "magit-%s-section-map" (progn (or ... ...) (aref it 1))))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or (and (vectorp it) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref it 3))) (while (< (point) end) (let ((next (or ... end))) (if (get-text-property (point) (quote magit-section)) nil (put-text-property (point) next (quote magit-section) it) (if map (progn ...))) (goto-char next))))) (if (eq it magit-root-section) (magit-section-show it) (progn (or (and (vectorp (progn (or ... ...) (aref it 11))) (>= (length (progn ... ...)) 13) (memq (aref (progn ... ...) 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v (progn (or ... ...) (aref it 11)))) (aset v 12 (nconc (progn (or ... ...) (aref ... 12)) (list it)))))))
  (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section (setq magit-root-section it)))))) (catch (quote cancel-section) (if branch nil (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-type) (quote magit-section))) (let* ((v it)) (aset v 1 (quote commit))))) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash (quote face) (quote magit-hash)) " " (and branch (concat (propertize branch (quote face) (quote magit-branch-local)) " ")) msg "\n") (if (or upstream (setq upstream (magit-get-tracked-branch branch))) (progn (setq line (or (magit-rev-format "%h %s" upstream) "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82716 line)) (let ((hash ...) (msg ...)) (let* (...) (progn ... ...) (let ... ... section82717)))))) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref it 3)) t) (let* ((end (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 5 ...)))) (map (intern (format "magit-%s-section-map" (progn ... ...)))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (aref it 3))) (while (< (point) end) (let ((next ...)) (if (get-text-property ... ...) nil (put-text-property ... next ... it) (if map ...)) (goto-char next))))) (if (eq it magit-root-section) (magit-section-show it) (progn (or (and (vectorp (progn ... ...)) (>= (length ...) 13) (memq (aref ... 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v (progn ... ...))) (aset v 12 (nconc (progn ... ...) (list it))))))) it)
  (let* ((it (progn (vector (quote cl-struct-magit-section) (quote branch) (or branch hash) (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v it)) (aset v 6 (let ((value (run-hook-with-args-until-success ... it))) (if value (eq value (quote hide)) (let (...) (if it ... nil))))))) (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section (setq magit-root-section it)))))) (catch (quote cancel-section) (if branch nil (progn (or (and (vectorp it) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-type) (quote magit-section))) (let* ((v it)) (aset v 1 (quote commit))))) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash (quote face) (quote magit-hash)) " " (and branch (concat (propertize branch (quote face) (quote magit-branch-local)) " ")) msg "\n") (if (or upstream (setq upstream (magit-get-tracked-branch branch))) (progn (setq line (or (magit-rev-format "%h %s" upstream) "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82716 line)) (let (... ...) (let* ... ... ...))))) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn (or (and (vectorp it) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref it 3)) t) (let* ((end (progn (or ... ...) (let* ... ...))) (map (intern (format "magit-%s-section-map" ...))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or ... ...) (aref it 3))) (while (< (point) end) (let (...) (if ... nil ... ...) (goto-char next))))) (if (eq it magit-root-section) (magit-section-show it) (progn (or (and (vectorp ...) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v ...)) (aset v 12 (nconc ... ...)))))) it))
  (let ((hash (match-string 1 string82715)) (msg (match-string 2 string82715))) (let* ((it (progn (vector (quote cl-struct-magit-section) (quote branch) (or branch hash) (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v it)) (aset v 6 (let ((value ...)) (if value (eq value ...) (let ... ...)))))) (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section ...))))) (catch (quote cancel-section) (if branch nil (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 1 ...)))) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash (quote face) (quote magit-hash)) " " (and branch (concat (propertize branch ... ...) " ")) msg "\n") (if (or upstream (setq upstream (magit-get-tracked-branch branch))) (progn (setq line (or ... "")) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let (...) (let ... ...)))) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (aref it 3)) t) (let* ((end (progn ... ...)) (map (intern ...)) (map (and ... ...))) (save-excursion (goto-char (progn ... ...)) (while (< ... end) (let ... ... ...)))) (if (eq it magit-root-section) (magit-section-show it) (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 12 ...))))) it)))
  (let ((string82715 line)) (let ((hash (match-string 1 string82715)) (msg (match-string 2 string82715))) (let* ((it (progn (vector (quote cl-struct-magit-section) (quote branch) (or branch hash) (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp it) (>= (length it) 13) (memq (aref it 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v it)) (aset v 6 (let (...) (if value ... ...))))) (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil ...)))) (catch (quote cancel-section) (if branch nil (progn (or ... ...) (let* ... ...))) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash (quote face) (quote magit-hash)) " " (and branch (concat ... " ")) msg "\n") (if (or upstream (setq upstream ...)) (progn (setq line ...) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ... ...))) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn (or ... ...) (aref it 3)) t) (let* ((end ...) (map ...) (map ...)) (save-excursion (goto-char ...) (while ... ...))) (if (eq it magit-root-section) (magit-section-show it) (progn (or ... ...) (let* ... ...)))) it))))
  (let ((line (magit-rev-format "%h %s" "HEAD"))) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82715 line)) (let ((hash (match-string 1 string82715)) (msg (match-string 2 string82715))) (let* ((it (progn (vector ... ... ... ... nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp it) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v it)) (aset v 6 (let ... ...)))) (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot (or magit-insert-section--oldroot ...))) (catch (quote cancel-section) (if branch nil (progn ... ...)) (magit-insert-heading (magit-string-pad "Head: " 10) (propertize hash ... ...) " " (and branch ...) msg "\n") (if (or upstream ...) (progn ... ... ...)) (run-hooks (quote magit-status-headers-hook)) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type (progn ... ...) t) (let* (... ... ...) (save-excursion ... ...)) (if (eq it magit-root-section) (magit-section-show it) (progn ... ...))) it)))))
  (if hash (let ((line (magit-rev-format "%h %s" "HEAD"))) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82715 line)) (let ((hash (match-string 1 string82715)) (msg (match-string 2 string82715))) (let* ((it (progn ...))) (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 6 ...))) (let ((magit-insert-section--current it) (magit-insert-section--parent it) (magit-insert-section--oldroot ...)) (catch (quote cancel-section) (if branch nil ...) (magit-insert-heading ... ... " " ... msg "\n") (if ... ...) (run-hooks ...) (insert "\n") (magit-insert-child-count it) (set-marker-insertion-type ... t) (let* ... ...) (if ... ... ...)) it))))) (insert "In the beginning there was darkness\n\n"))
  (let ((hash (magit-rev-verify "HEAD"))) (if hash (let ((line (magit-rev-format "%h %s" "HEAD"))) (string-match "^\\([^ ]+\\) \\(.*\\)" line) (let ((string82715 line)) (let ((hash (match-string 1 string82715)) (msg (match-string 2 string82715))) (let* ((it ...)) (progn (or ... ...) (let* ... ...)) (let (... ... ...) (catch ... ... ... ... ... ... ... ... ... ...) it))))) (insert "In the beginning there was darkness\n\n")))
  magit-insert-status-headers()
  run-hooks(magit-status-sections-hook)
  (catch (quote cancel-section) (run-hooks (quote magit-status-sections-hook)) (magit-insert-child-count section82714) (set-marker-insertion-type (progn (or (and (vectorp section82714) (>= (length section82714) 13) (memq (aref section82714 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref section82714 3)) t) (let* ((end (progn (or (and (vectorp section82714) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-end) (quote magit-section))) (let* ((v section82714)) (aset v 5 (point-marker))))) (map (intern (format "magit-%s-section-map" (progn (or ... ...) (aref section82714 1))))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or (and (vectorp section82714) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref section82714 3))) (while (< (point) end) (let ((next (or ... end))) (if (get-text-property (point) (quote magit-section)) nil (put-text-property (point) next (quote magit-section) section82714) (if map (progn ...))) (goto-char next))))) (if (eq section82714 magit-root-section) (magit-section-show section82714) (progn (or (and (vectorp (progn (or ... ...) (aref section82714 11))) (>= (length (progn ... ...)) 13) (memq (aref (progn ... ...) 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v (progn (or ... ...) (aref section82714 11)))) (aset v 12 (nconc (progn (or ... ...) (aref ... 12)) (list section82714)))))))
  (let ((magit-insert-section--current section82714) (magit-insert-section--parent section82714) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section (setq magit-root-section section82714)))))) (catch (quote cancel-section) (run-hooks (quote magit-status-sections-hook)) (magit-insert-child-count section82714) (set-marker-insertion-type (progn (or (and (vectorp section82714) (>= (length section82714) 13) (memq (aref section82714 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref section82714 3)) t) (let* ((end (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (let* (...) (aset v 5 ...)))) (map (intern (format "magit-%s-section-map" (progn ... ...)))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or (and ... ... ...) (error "%s accessing a non-%s" ... ...)) (aref section82714 3))) (while (< (point) end) (let ((next ...)) (if (get-text-property ... ...) nil (put-text-property ... next ... section82714) (if map ...)) (goto-char next))))) (if (eq section82714 magit-root-section) (magit-section-show section82714) (progn (or (and (vectorp (progn ... ...)) (>= (length ...) 13) (memq (aref ... 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v (progn ... ...))) (aset v 12 (nconc (progn ... ...) (list section82714))))))) section82714)
  (let* ((section82714 (progn (vector (quote cl-struct-magit-section) (quote status) nil (point-marker) nil nil nil nil nil nil nil magit-insert-section--parent nil)))) (progn (or (and (vectorp section82714) (>= (length section82714) 13) (memq (aref section82714 0) cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-hidden) (quote magit-section))) (let* ((v section82714)) (aset v 6 (let ((value (run-hook-with-args-until-success ... section82714))) (if value (eq value (quote hide)) (let (...) (if it ... nil))))))) (let ((magit-insert-section--current section82714) (magit-insert-section--parent section82714) (magit-insert-section--oldroot (or magit-insert-section--oldroot (if magit-insert-section--parent nil (prog1 magit-root-section (setq magit-root-section section82714)))))) (catch (quote cancel-section) (run-hooks (quote magit-status-sections-hook)) (magit-insert-child-count section82714) (set-marker-insertion-type (progn (or (and (vectorp section82714) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-start) (quote magit-section))) (aref section82714 3)) t) (let* ((end (progn (or ... ...) (let* ... ...))) (map (intern (format "magit-%s-section-map" ...))) (map (and (boundp map) (symbol-value map)))) (save-excursion (goto-char (progn (or ... ...) (aref section82714 3))) (while (< (point) end) (let (...) (if ... nil ... ...) (goto-char next))))) (if (eq section82714 magit-root-section) (magit-section-show section82714) (progn (or (and (vectorp ...) (>= ... 13) (memq ... cl-struct-magit-section-tags)) (error "%s accessing a non-%s" (quote magit-section-children) (quote magit-section))) (let* ((v ...)) (aset v 12 (nconc ... ...)))))) section82714))
  magit-status-refresh-buffer()
  apply(magit-status-refresh-buffer nil)
  (save-excursion (apply magit-refresh-function magit-refresh-args))
  (let ((inhibit-read-only t)) (erase-buffer) (save-excursion (apply magit-refresh-function magit-refresh-args)))
  (let* ((buffer (current-buffer)) (windows (apply (quote append) (mapcar (function (lambda (it) (let ... ...))) (or (get-buffer-window-list buffer nil t) (list (selected-window))))))) (deactivate-mark) (setq magit-section-highlight-overlays nil magit-section-highlighted-sections nil) (let ((inhibit-read-only t)) (erase-buffer) (save-excursion (apply magit-refresh-function magit-refresh-args))) (let ((--dolist-tail-- windows) window) (while --dolist-tail-- (setq window (car --dolist-tail--)) (let ((save-selected-window--state (internal--before-with-selected-window (car window)))) (save-current-buffer (unwind-protect (progn (select-window ... ...) (save-current-buffer ... ...)) (internal--after-with-selected-window save-selected-window--state)))) (setq --dolist-tail-- (cdr --dolist-tail--)))) (run-hooks (quote magit-refresh-buffer-hook)) (magit-section-update-highlight))
  (progn (let* ((buffer (current-buffer)) (windows (apply (quote append) (mapcar (function (lambda ... ...)) (or (get-buffer-window-list buffer nil t) (list ...)))))) (deactivate-mark) (setq magit-section-highlight-overlays nil magit-section-highlighted-sections nil) (let ((inhibit-read-only t)) (erase-buffer) (save-excursion (apply magit-refresh-function magit-refresh-args))) (let ((--dolist-tail-- windows) window) (while --dolist-tail-- (setq window (car --dolist-tail--)) (let ((save-selected-window--state (internal--before-with-selected-window ...))) (save-current-buffer (unwind-protect (progn ... ...) (internal--after-with-selected-window save-selected-window--state)))) (setq --dolist-tail-- (cdr --dolist-tail--)))) (run-hooks (quote magit-refresh-buffer-hook)) (magit-section-update-highlight)))
  (if magit-refresh-function (progn (let* ((buffer (current-buffer)) (windows (apply (quote append) (mapcar (function ...) (or ... ...))))) (deactivate-mark) (setq magit-section-highlight-overlays nil magit-section-highlighted-sections nil) (let ((inhibit-read-only t)) (erase-buffer) (save-excursion (apply magit-refresh-function magit-refresh-args))) (let ((--dolist-tail-- windows) window) (while --dolist-tail-- (setq window (car --dolist-tail--)) (let ((save-selected-window--state ...)) (save-current-buffer (unwind-protect ... ...))) (setq --dolist-tail-- (cdr --dolist-tail--)))) (run-hooks (quote magit-refresh-buffer-hook)) (magit-section-update-highlight))))
  magit-refresh-buffer()
  magit-mode-init("/<repo-path>" magit-status-mode magit-status-refresh-buffer)
  apply(magit-mode-init "/<repo-path>/" (magit-status-mode magit-status-refresh-buffer))
  (save-current-buffer (set-buffer buf-symb82713) (apply (function magit-mode-init) toplevel82711 init-args82712))
  (if toplevel82711 (save-current-buffer (set-buffer buf-symb82713) (apply (function magit-mode-init) toplevel82711 init-args82712)) (user-error "Not inside a Git repository"))
  (let* ((mode-symb82710 (function magit-status-mode)) (toplevel82711 (magit-get-top-dir)) (init-args82712 (list mode-symb82710 (function magit-status-refresh-buffer))) (buf-symb82713 (magit-mode-display-buffer magit-status-buffer-name-format mode-symb82710 (or switch-function magit-status-buffer-switch-function)))) (if toplevel82711 (save-current-buffer (set-buffer buf-symb82713) (apply (function magit-mode-init) toplevel82711 init-args82712)) (user-error "Not inside a Git repository")))
  (let ((default-directory directory)) (let* ((mode-symb82710 (function magit-status-mode)) (toplevel82711 (magit-get-top-dir)) (init-args82712 (list mode-symb82710 (function magit-status-refresh-buffer))) (buf-symb82713 (magit-mode-display-buffer magit-status-buffer-name-format mode-symb82710 (or switch-function magit-status-buffer-switch-function)))) (if toplevel82711 (save-current-buffer (set-buffer buf-symb82713) (apply (function magit-mode-init) toplevel82711 init-args82712)) (user-error "Not inside a Git repository"))))
  magit-status-internal("/<repo dir>/")
  (if directory (let ((toplevel (magit-toplevel-safe directory))) (setq directory (file-name-as-directory (expand-file-name directory))) (if (and toplevel (string-equal directory toplevel)) (magit-status-internal directory) (if (y-or-n-p (if toplevel (format "%s is a repository.  Create another in %s? " toplevel directory) (format "Create repository in %s? " directory))) (progn (magit-init directory))))) (magit-status-internal default-directory))
  magit-status(nil)
  call-interactively(magit-status nil nil)
  command-execute(magit-status)
```

**Comment by totallymike** [comment]
For the record, it only happens on this particular repo, as far as I can tell.

**Comment by Silex** [comment]
@totallymike: I had this the other day... it's because your local branch has an upstream that was deleted.

Try removing the upstream or make sure it's correct and it'd go away.

**Comment by Silex** [comment]
There's a patch available in #1723
